firebase/BUILD.bazel

```diff
@@ -0,0 +1,30 @@
+''' Builds and exports Firebase dependencies.
+To reference these dependencies, add '//firebase:crashlytics_lib' and '//firebase:crashlytics_deps'
+to your build rule's dependency list.
+'''
+load("@rules_jvm_external//:defs.bzl", "artifact")
+load("@dagger//:workspace_defs.bzl", "dagger_rules")
+load("@tools_android//tools/crashlytics:defs.bzl", "crashlytics_android_library")
+load("@tools_android//tools/googleservices:defs.bzl", "google_services_xml")
+
+package(default_visibility = ["//visibility:public"])
+
+GOOGLE_SERVICES_RESOURCES = google_services_xml(
+    package_name = "org.oppia.app",
+    google_services_json = "google-services.json",
+)
+
+crashlytics_android_library(
+    name = "crashlytics_lib",
+    package_name = "org.oppia.app",
+    build_id = "48fc9d17-e102-444c-8e0d-638d75ec0942",
```

Comment:
What is this?

Reply:
According to https://github.com/bazelbuild/tools_android/blob/master/tools/crashlytics/README.md#firebase-crashlytics-support-with-bazel a build-id is needed. Not exactly sure why.

Reply:
Ah, thanks for the clarification.